### PR TITLE
Index: added max size limit

### PR DESCRIPTION
**Pull request by bryanlatten**
- Updated volume language
- Colorized output

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -92,11 +95,27 @@ getStdin().then(str => {
...
   var failedVolumes = false;
   var failedPortRequirement = false;
   var failedPortRange = false;
+  var failedMaxSize = false;
+
+  if (maxSize !== null ) {
+    if (containerSize > maxSize) {
+      failedMaxSize = true;
+      console.log("[FAIL] %dMB exceeded %dMB container size limit", containerSize, maxSize);
+    } else {
```
Comment (chr0n1x) [inline index.js]: no [optional] warning threshold?

Comment (bryanlatten) [reply index.js]: good suggestion

**Comment by bossjones** [comment]
👍

**Comment by bossjones** [comment]
color output yay!

File: index.js [hunk 2]
```diff
@@ -92,11 +95,51 @@ getStdin().then(str => {
...
+
   var failedEnvKeys = [];
   var failedLabels = [];
   var failedVolumes = false;
   var failedPortRequirement = false;
   var failedPortRange = false;
+  var failedMaxSize = false;
+  var failedWarningSize = false;
+
+  if (warningSize !== null && containerSize >= warningSize) {
```
Comment (chr0n1x) [inline index.js]: lmao omg `if (bool condition is true) { boo_var = true; }`
You're making [this dude](https://www.binghamton.edu/cs/people/lander.html) cry.

Comment (chr0n1x) [reply index.js]: he threatened my class that if anyone ever did this he'd fail them

File: index.js [hunk 3]
```diff
@@ -92,11 +95,51 @@ getStdin().then(str => {
...
+    failedWarningSize = true;
+  }
+
+  if (maxSize === null ) {
+
+    if (failedWarningSize) {
+      console.log("[%s] %dMB container size should be below %dMB", clc.yellow('WARN'), containerSize, warningSize);
```
Comment (chr0n1x) [inline index.js:119]: I don't understand why not just log right outside of `if (maxSize...)` instead of duplicating twice

Comment (bryanlatten) [reply index.js:119]: its a different message

File: index.js (hunk 2 above)
Comment (bryanlatten) [reply index.js]: fixed :)

Comment (bryanlatten) [reply index.js]: that's what happens when you code during meetings

**Comment by chr0n1x** [comment]
👍

**Pull request merged**